masq_lib/src/shared_schema.rs

```diff
@@ -526,9 +526,110 @@ impl ConfiguratorError {
+     a configuration with multiple clandestine ports.\n\n\
+     If you have more than one descriptor, separate them with commas. Whether single or multiple descriptors, they \
+     should be enclosed by quotes. No default value is available; \
+     if you don't specify a neighbor, your Node will start without being connected to any MASQ \
+     Network, although other Nodes will be able to connect to yours if they know your Node's descriptor. \
+     --neighbors is meaningless in --neighborhood-mode zero-hop.");
+        assert_eq!(NEIGHBORHOOD_MODE_HELP, "This configures the way the Node relates to other Nodes.\n\n\
+     zero-hop means that your Node will operate as its own MASQ Network and will not communicate with any \
+     other Nodes. --ip, --neighbors, and --clandestine-port are incompatible with --neighborhood_mode \
+     zero-hop.\n\n\
+     originate-only means that your Node will not accept connections from any other Node; it \
+     will only originate connections to other Nodes. This will reduce your Node's opportunity to route \
+     data (it will only ever have two neighbors, so the number of routes it can participate in is limited), \
+     it will reduce redundancy in the MASQ Network, and it will prevent your Node from acting as \
+     a connection point for other Nodes to get on the Network; but it will enable your Node to operate in \
+     an environment where your network hookup is preventing you from accepting connections, and it means \
+     that you don't have to forward any incoming ports through your router. --ip and --clandestine_port \
+     are incompatible with --neighborhood_mode originate-only.\n\n\
+     consume-only means that your Node will not accept connections from or route data for any other Node; \
+     it will only consume services from the MASQ Network. This mode is appropriate for devices that \
+     cannot maintain a constant IP address or stay constantly on the Network. --ip and --clandestine_port \
+     are incompatible with --neighborhood_mode consume-only.\n\n\
+     standard means that your Node will operate fully unconstrained, both originating and accepting \
+     connections, both consuming and providing services, and when you operate behind a router, it \
+     requires that you forward your clandestine port through that router to your Node's machine.");
+        assert_eq!(MAPPING_PROTOCOL_HELP, "The Node can speak three protocols to your router to make it allow outside Nodes to connect inward \
+    through it to your machine. These three protocols are pcp, pmp, and igdp. The Node can try them one \
+    by one to determine which your router supports, but if you happen to know already, you can supply the \
+    name of the protocol here. If you've taken care of port mapping in some other way, \
+    and you don't need Node to negotiate with your router, say 'none' here and be sure to specify your \
```

Comment:
You could put a single space in between the left border and the start of the text.